docker-compose.yml

```diff
@@ -1,5 +1,4 @@
 ---
-version: "2"
```

Comment:
fixes:
```
WARN[0000] /Users/relud/socorro/docker-compose.yml: the attribute `version` is obsolete, it will be ignored, please remove it to avoid potential confusion
```